flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -907,19 +900,19 @@ private ApplicationReport startAppMaster(
 		BootstrapTools.writeConfiguration(configuration, tmpConfigurationFile);
 
 		String flinkConfigKey = "flink-conf.yaml";
-		Path remotePathConf = setupSingleLocalResource(
+		YarnLocalResourceDescription localResourceDescConf = setupSingleLocalResource(
```

Comment:
makes naming consistent? maybe: `localResourceDescFlinkConf `?

Reply:
Make sense to me. I will update.